### ISSUE TITLE
Desktop Frontend

### ISSUE THREAD

**Issue by LeoMattosMartins** [MakeyDooey/Software#1]
**Feature Member(s):** @KoenLin (Feature Lead), @LeoMattosMartins

**Feature Repo(s):** Software

**Acceptance Criteria:**

Dashboard Layout

- [ ] User can add/move/remove widgets: this change should persist in a dashboard_config.JSON locally which can be exported and imported 
- [ ] The dashboard should be able to display graphs and other tracking information generated by the platform 

Widgets

- [ ] Buttons, sliders, etc... can be bound to script hooks or other custom commands (this allows us to be flexible in the type of app we want to make) 
- [ ] Widgets should have associated tool tips

Accessibility Features

- [ ] WCAG 2.2 Level AA Compliance 
- [ ] Support for some library like react-itn or i18next for internationalization (tentative)

Logging

- [ ] Log panel with history and customizable message filtering
- [ ] Logs can be exported as a: .JSON, .CSV, .TSV, and .SCSV 

**Dependencies:**
- Figma 
- Defining the type of app
